testsuite/MDAnalysisTests/analysis/test_encore.py

```diff
@@ -125,10 +125,10 @@ def test_triangular_matrix(self):
                        reason="Not yet supported on Windows.")
     def test_parallel_calculation(self):
 
-        arguments = [tuple([i]) for i in np.arange(0,100)]
+        arguments = [tuple([i]) for i in np.arange(0,10)]
 
         parallel_calculation = encore.utils.ParallelCalculation(function=function,
-                                                                n_jobs=4,
+                                                                n_jobs=2,
```

Comment:
good catch!!

Reply:
I assume that this change alone (together with restricting threads) accounts for a lot of improvement.